tests/test_angles.py

```diff
@@ -43,3 +43,10 @@ def test_generator():
     a = md.compute_angles(ptraj, triplets)
     b = md.compute_angles(ptraj, triplets2)
     eq(a, b)
+
+def test_angle_180(get_fn):
+    # Test that linear coordinates yield 180 degrees, not "nan"
+    a = md.load(get_fn('mol.gro'))
+    b = md.compute_angles(a, [[0, 1, 2]])
+    assert not np.isnan(b[0][0])
```

Comment:
If this angle is meant to be 180 / pi, could we assert that as well? (Or that it's close to pi - `pytest.approx` is one way to do this)

Reply:
Thought I'd replied to this but obviously not: good idea, thanks for the suggestion, I've added the extra test